drivers/hmis/app/models/hmis/form/instance.rb

```diff
@@ -0,0 +1,12 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Form::Instance < ApplicationRecord
+  self.table_name = :hmis_form_instances
+
+  belongs_to :entity, polymorphic: true, optional: true
+  belongs_to :definition, foreign_key: :identifier, primary_key: :form_definition_identifier
```

Comment:
Should it be `definition_identifier` instead of `form_definition_identifier`? I'm still shaky on my rails associations. But I don't see anything called `form_definition_identifier`. In any case, I think also fine to merge this and leave the model/relation stuff up to Zach!

Reply:
The DB column is `form_definition_identifier` so we have to specify it.  I'm not 100% sure I got the primary/foreign key naming correct, but it should be quick to fix if I didn't.